examples/sample-app/README.md

```diff
@@ -104,7 +104,21 @@ the present working directory is the same directory as this README.
         $ export CURL_CA_BUNDLE=`pwd`/openshift.local.certificates/admin/root.crt
         $ sudo chmod +r "$KUBECONFIG"
 
-4. Deploy a private docker registry within OpenShift with the certs necessary for access to master:
+4. Bind a user to the `view` role in the default namespace so you can observe progress in the web console
+
+        $ openshift ex policy add-user view anypassword:test-admin
+
+5. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8444`.  Login using user, `test-admin` and any password.
```

Comment:
This step is labelled optional.  You can look at two times.  Once to see the docker registry deployed and once to see the rest of the test resources created.